emgena/serializers.py

```diff
@@ -84,6 +98,8 @@ def create(self, validated_data):
             "Subject": n.subject,
             "Text": n.message.replace("\n", ';')
         }
+        if n.cc != '' and n.cc is not None:
```

Comment:
```suggestion
        if n.cc:
```
(`''` and `None` are both falsy)`

Reply:
TIL you can do that in python!
Thanks